Presentation.Web/app/components/data-processing/tabs/data-processing-registration-tab-main.controller.ts

```diff
@@ -244,7 +253,15 @@
                 (query) => this
                     .dataProcessingRegistrationService
                     .getApplicableSubDataProcessors(this.dataProcessingRegistrationId, query, pageSize)
-                    .then(results => this.mapDataProcessingSearchResults(results))
+                    .then(results => this.mapDataProcessingSearchResults(results)),
+                null,
+                newElement => {
+                    var result = `<div>${newElement.text}</div>`;
```

Comment:
this code block is duplicated in line 236->240. Let's make a mapping function for it so it can be reused